spec/quickbooks-payments/basic_model_spec.rb

```diff
@@ -20,7 +25,18 @@ def _fakekey=(val)
   end
 
   describe 'instance methods' do
-    let(:options) { { "token" => "abcdef" } }
-    subject(:instance) { described_class.new options }
+    let(:options) { { 'token' => 'abcdef' } }
+    let(:instance) { described_class.new options }
+    let(:args) { [] }
+
+    subject(:run) { instance.send method, *args }
+
+    describe 'to_json' do
+      let(:method) { 'to_json' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.